### ISSUE TITLE
Windows tray menu issues: Flickering and no \n in status text

### ISSUE THREAD

**Issue by binwiederhier** [syncany/syncany#337]
The Windows tray menu and status text have a two issues:

**(1) Flickering:**
Each update and status text change currently causes the entire menu to be rebuilt. While Linux handles that very well, Windows apparently has issues with that. Code in question:

```java
Display.getDefault().asyncExec(new Runnable() {
	public void run() {
		if (root != null) {
			statusTexts.put(root, statusText);
		}
		else {
			statusTexts.clear();
		}
		
		buildMenuItems();
	}
});
```

Instead of rebuilding, we should maybe only (a) add/remove status text items when needed or (b) hide the status items from 'in sync' watches, or (c) in the worst case, reduce the per-folder status texts to a single status item (in the Windows case).

**(2) No \n**
The Linux tray status text shows a multi-line status text. The first line is the watch name, the second line the status. Windows doesn't support \n in a tray item. We need to fix that for Windows.

**Comment by binwiederhier** [comment]
Both of those are fixed as part of #332. The changes are merge-ready. I'll merge them after @pimotte's review on the weekend.

**Comment by binwiederhier** [comment]
Fixed with #332
